EpiserverRedirects/Extensions/DependencyInjection/ContentProviderRegistrationExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Forte.EpiserverRedirects.Configuration;
+using Forte.EpiserverRedirects.Resolver.Content;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Forte.EpiserverRedirects.Extensions.DependencyInjection;
+
+public static class ContentProviderRegistrationExtensions
+{
+    public static IServiceCollection AddContentProviderResolvers(this IServiceCollection services)
```

Comment:
I think it would be best if this method is called in `ConfigureEpiserverRedirects` method? Otherwise every project that is using this package would have to call this method by its own?

Reply:
Yeah. Firstly I wanted to add options to for example add Commerce Nuget package with definition of CommerceContentProviderResolver but there were some problems with auto-detecting them. 
I should be fine to register both of them.